src/mechanics/elasticity.cc

```diff
@@ -473,43 +567,49 @@ void Elasticity::next_time()
 void Elasticity::solve_linear_system()
 {
     START_TIMER("data reinit");
-    data_.set_time(time_->step(), LimitSide::right);
+    eq_fields_->set_time(time_->step(), LimitSide::right);
     END_TIMER("data reinit");
     
     // assemble stiffness matrix
     if (stiffness_matrix == NULL
-        || data_.subset(FieldFlag::in_main_matrix).changed())
+        || eq_fields_->subset(FieldFlag::in_main_matrix).changed())
     {
         DebugOut() << "Mechanics: Assembling matrix.\n";
-        ls->start_add_assembly();
-        ls->mat_zero_entries();
-        assemble_stiffness_matrix();
-        ls->finish_assembly();
+        eq_data_->ls->start_add_assembly();
+        eq_data_->ls->mat_zero_entries();
+        START_TIMER("assemble_stiffness");
```

Comment:
Move this timer into the assemble method? As the GenericAssembly is 
specialized we can take the FRAME name from the Assemble<> template.

Reply:
Will be solved together with DG after merge.